agent/util-scripts/pbench-tool-meister-start

```diff
@@ -0,0 +1,450 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+"""pbench-tool-meister-start
+
+Responsible for:
+
+   1. Starting a redis server
+   2. Loading up the tool group data for the target group into the redis server
+   3. Starting the tool-data-sink process
+   4. Starting all the local and remote tool meisters
+
+When complete we leave running locally, a redis server and a tool data sink
+process, and any local or remote tool meisters.
+
+The pbench-tool-meister-stop will take care of (gracefully) stopping all of
+these process, locally or remotely.
+"""
+
+import sys
+import json
+import os
+import time
+import errno
+import redis
+import logging
+
+# Port number is "One Tool" in hex 0x17001
+# FIXME: move to common area
+redis_port = 17001
+
+# Redis server configuration template for pbench's use
```

Comment:
This should be in an pbench-agent.cfg IMHO

Reply:
Not sure how that would work.  We could use a Jinja2 template to do this, but given it is so small and Python offers that cleanly, this seemed a solid supportable approach.

We also don't want to invite the users to control how the redis instance runs to avoid conflicts with our required operation of it.